daffodil-cli/src/it/scala/org/apache/daffodil/executing/TestCLIexecuting.scala

```diff
@@ -162,6 +175,8 @@ class TestCLIexecuting {
       val cmd = String.format("%s test %s byte_entities_6_08", Util.binPath, testTdmlFile)
       shell.sendLine(cmd)
       shell.expect(contains("[Pass] byte_entities_6_08"))
+
+      Util.expectExitCode(ExitCode.TestError, shell)
```

Comment:
Here's another example. We are running a TDML test, it's passing, yet we are checking for TestError. I would expect this to be a Success.